lib/Admin.php

```diff
@@ -21,7 +21,7 @@ public static function init() {
 	public static function meta_links( $links, $file ) {
 		if ( strstr($file, '/timber.php') ) {
 			unset($links[2]);
-			$links[] = '<a href="/wp-admin/plugin-install.php?tab=plugin-information&amp;plugin=timber-library&amp;TB_iframe=true&amp;width=600&amp;height=550" class="thickbox" aria-label="More information about Timber" data-title="Timber">View details</a>';
+			$links[] = '<a href="' . get_site_url(). '/wp-admin/plugin-install.php?tab=plugin-information&amp;plugin=timber-library&amp;TB_iframe=true&amp;width=600&amp;height=550" class="thickbox" aria-label="More information about Timber" data-title="Timber">View details</a>';
```

Comment:
Could we also use `admin_url( 'plugin-install.php…' )` instead of `get_site_url( '/wp-admin/plugin-install.php…' )`?

Reply:
I think that one is even better...  Didn't think of that one... I will update my PR.